src/_pytest/stepwise.py

```diff
@@ -8,7 +13,12 @@
 from _pytest.reports import TestReport
 
 
-STEPWISE_CACHE_DIR = "cache/stepwise"
+if TYPE_CHECKING:
+    from typing import ClassVar
+
+    from typing_extensions import Self
+
+STEPWISE_CACHE_DIR = "cache/stepwise2"
```

Comment:
Using a different cache key because the cache format has changed.

Reply:
I wonder if that's really necessary? The main drawback I can see is that it would leave stale data around from the old key, I think? Given that we do handle data in invalid formats below, wouldn't it just complain once about the old "invalid" data being there (which would only be the case if pytest was run with `--stepwise` immediately before upgrading it), and then that would be discarded and we're good?